guides/common/attributes-base.adoc

```diff
@@ -13,21 +13,49 @@
+// :RepoRHEL7ServerOptional: any equivalent for Sat7 for RHEL 8? - "codeready-builder-for-rhel-8-x86_64-rpms" ?
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
-// For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.10-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.10-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL8ServerAnsible: ansible-2.9-for-rhel-8-x86_64-rpms 
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.10-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-7.0-rpms
+:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-7.0-for-rhel-8-x86_64-rpms
+// Satellite-Tools will become Satellite-Client ??? - I haven't found this repo mentioned anywhere except for this file. So it may not be needed. The below was mentioend in 6.10 docs.
+// :RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
+// Repos relevant for Puppet - The below was mentioend for 6.10 docs.
 // Do not update the puppet4 repo versions. They must stay at 6.3.
-:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
-:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+// :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
+// :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+
+//Added by Marie for Beta
+// Repositories and subscriptions (used both in Katello and Satellite guides)
+//:SatelliteSub: Red Hat Satellite Infrastructure Subscription
+//:RepoRHEL7Server: rhel-7-server-satellite
+//:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms 
+// We would probably need to mention also "rhel-8-for-x86_64-appstream-rpms"
+//:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
+//:RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
+//:RepoRHEL7ServerOptional: any equivalent for Sat7 for RHEL 8? - "codeready-builder-for-rhel-8-x86_64-rpms" ?
+//:RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
+//:RepoRHEL8ServerAnsible: ansible-2.9-for-rhel-8-x86_64-rpms 
+//:RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-7-beta-rpms
```

Comment:
What I would suggest is
```suggestion
// Set to beta- when this version is in beta
:RepoSatelliteBeta: beta-
:RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-7-{RepoSatelliteBeta}rpms
```

Then when the beta is over, you simply empty the `RepoSatelliteBeta` variable and there's no need to modify all the other macros.

Reply:
Thank you, @ekohl ! LGTM , but actually I'm not sure whether this would work. All beta repos include just "7"  (not 7.0) and the {ProductVersion} needs to be set to "7.0".  Also, the GA repos mostly include 7.0, so the automatic replacement would maybe require some more new attributes.

Reply:
I recall we still had some discussion about 7.0 vs 7 naming in repo names. @ehelms did we make a decision on that already?

Reply:
Also, looking more closely, it's really `7.0` vs `7-beta`. So you could also name it `RepoSatelliteVersion` and use that instead. My point is that you'd want a single place to modify it so the change from beta -> non-beta is as small as it can be.

Reply:
Wouldn't this work for both cases?
```suggestion
// For beta
:RepoSatelliteVersion: 7-beta
// For a release
:RepoSatelliteVersion: {ProductVersion}

:RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-{RepoSatelliteVersion}-rpms
```

Reply:
@ekohl, nice and easy! Thank you! :-) I applied it. However, this are couple more issues. While GA repos for RHEL 7 in all cases use "7-server", the Beta repos mostly use "server-7". So I introduced one more attribute to resolve this. However, the names of Beta repos in the source document are even not consistent - some use "server-7" and some other "7-server". I assume this is just a mistake in the source document, and all Beta repos should be called consistently. I've already asked that in the source document with repo IDs. If it's just a mistake, and all Beta repos should be named consistently, then my current solution should work.  If it's not mistake and Beta repo IDs are really inconsistent, then we need to go back to the previous solution - so two blocks of text - one that would apply for Beta and another one for GA.